ochami-cli

```diff
@@ -281,67 +507,81 @@ def main():
                 access_token = f.read()
                 if access_token:
                     headers = {"Authorization": f"Bearer {access_token}"}
+    if not checkTokenExpired(access_token):
+        sys.exit(1)
```

Comment:
Resolved with [371bf1f](https://github.com/OpenCHAMI/ochami-cmdline/pull/9/commits/371bf1f0e8bcb83cfa75af1abb6ae35e2ba6d8cc).

Reply:
We actually need to amend this. If using the `login` subcommand, we do not want to exit if there is no token because this command sets the token.

```diff
diff --git a/ochami-cli b/ochami-cli
index 272e585..841970e 100755
--- a/ochami-cli
+++ b/ochami-cli
@@ -509,11 +509,12 @@ def main():
                     headers = {"Authorization": f"Bearer {access_token}"}
 
     # Check Token
-    if not access_token:
-        print("ERROR: Neither --access-token, --access-token-file, nor ACCESS_TOKEN were set.")
-        sys.exit(1)
-    if not checkTokenExpired(access_token):
-        sys.exit(1)
+    if args.command != "login":
+        if not access_token:
+            print("ERROR: Neither --access-token, --access-token-file, nor ACCESS_TOKEN were set.")
+            sys.exit(1)
+        if not checkTokenExpired(access_token):
+            sys.exit(1)
     
     # set CA cert var
     if os.getenv("CACERT"):
```

Reply:
Resolved with [e852cf3](https://github.com/OpenCHAMI/ochami-cmdline/pull/9/commits/e852cf37d642a398529052e7cdd5bb4e4b920e79).

Reply:
Should be resolved by e852cf3